### DISCUSSION TITLE
OpenFAST - yield strength

### DISCUSSION THREAD

**Discussion by gabriellelarzilliere** [OpenFAST/openfast#2088] [General]
Hi, 
I have a question concerning the material properties of the NREL Offshore 5MW turbine of OpenFAST. Where could I find the values of the yield strengths of the turbine tower?

Thank you for the help.

**Comment by jjonkman** [comment]
Dear @gabriellelarzilliere,

The yield stress of the tower was never specified for the tower of the NREL 5-MW baseline wind turbine, but steel is often assumed to have a value around 250 MPa.

FYI: The following topic on our forum is likely of use to you: https://forums.nrel.gov/t/mlife-user-defined-distribution/1970.

Best regards,
